src/Components/Buttons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from './Button';
+import styles from 'styled-components';
+
+const Buttons = () => {
+
```

Comment:
This space here can be removed. Keep this in mind for the future PRs, don't change this one.